README.md

```diff
@@ -621,11 +640,64 @@ Divide.call(4, '2') #<BCDD::Result::Failure type=:invalid_arg value="arg2 must b
 
 ##### Important Requirement
 
-The unique condition for using the `#and_then` to call methods is that they must use the `Success()` and `Failure()` to produce their results.
+To use the `#and_then` method to call methods, they must use `Success()` and `Failure()` to produce the results.
+
+If you try to use `BCDD::Result::Subject()`/`BCDD::Result::Failure()`, or results from another `BCDD::Result::Mixin` instance with `#and_then`, it will raise an error because the subjects will be different.
```

Comment:
```suggestion
If you try to use `BCDD::Result::Subject()`/`BCDD::Result::Failure()`, or results from another `BCDD::Result.mixin` instance with `#and_then`, it will raise an error because the subjects will be different.
```